client/src/components/AddVideo.js

```diff
@@ -0,0 +1,97 @@
+import React,{ useState } from "react";
+
+const dateFormat = (date) => {
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+  return `${day}-${month}-${year}`;
+};
+
+const timeFormat = (date) => {
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const seconds = date.getSeconds();
+  return `${hours}-${minutes}-${seconds}`;
+};
```

Comment:
these functions work well! You could also use a method like `new Date().toLocaleString()` which is pretty useful

Reply:
done